dd-sdk-android-sqldelight/transitiveDependencies

```diff
@@ -6,11 +6,10 @@ com.squareup.okhttp3:okhttp:3.12.6                              :  413 Kb
 com.squareup.okio:okio:1.15.0                                   :   86 Kb
 com.squareup.sqldelight:android-driver:1.4.3                    :   22 Kb
 com.squareup.sqldelight:runtime-jvm:1.4.3                       :   42 Kb
-org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.3.72   :    8 Kb
-org.jetbrains.kotlin:kotlin-stdlib-common:1.4.10                :  186 Kb
+org.jetbrains.kotlin:kotlin-stdlib-common:1.4.20                :  186 Kb
 org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.0                   :    3 Kb
```

Comment:
indeed those are from `1.4.0` co-existing with `1.4.20`, but to resolve that update of `sqldelight` is needed. anyway, since it is still the same language level, it is fine.